pkg/bootstrap/docker/openshift/ansible.go

```diff
@@ -0,0 +1,308 @@
+package openshift
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"path"
+	"text/template"
+
+	"github.com/golang/glog"
+
+	kapierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kapi "k8s.io/kubernetes/pkg/api"
+	kbatch "k8s.io/kubernetes/pkg/apis/batch"
+	kclient "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
+
+	"github.com/openshift/origin/pkg/bootstrap/docker/errors"
+	"github.com/openshift/origin/pkg/bootstrap/docker/host"
+)
+
+const (
+	defaultAnsibleImageUser        = "root"
+	defaultOpenshiftAnsibleImage   = "ansible"
```

Comment:
@smarterclayton I don't know what your ask is here?  We parameterize the inventory file.  We additionally expect there to be an openshift ansible image that is tagged and prefixed like the other images (e.g. origin-ansible, ocp-ansible)